pkgs/development/perl-modules/findimagedupes/default.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper, perlPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "findimagedupes";
+  version = "2.19.1";
+
+  src = fetchFromGitHub {
+    owner = "jhnc";
+    repo = "findimagedupes";
+    rev = version;
+    sha256 = "sha256-KBRUlO0OpsrMv+ZKQBNrEw38ov+ijmpZ5vPg/7uCDKY=";
+  };
+
+  nativeBuildInputs = with perlPackages; [ makeWrapper PodMarkdown ];
+
+  propagatedBuildInputs = with perlPackages; [ perl ];
+
+  buildInputs = (with perlPackages; [
+    DBFile
+    FileMimeInfo
+    FileBaseDir
+    GraphicsMagick
+  ]);
```

```suggestion
  buildInputs = with perlPackages; [
    DBFile
    FileMimeInfo
    FileBaseDir
    GraphicsMagick
  ];
```

those are optional and only required when adding two lists.